### PR TITLE
LPS-67567 Updating the textarea's rows attribute when the field value change

### PR DESCRIPTION
**Pull request by thiago-rocha**

### CODE REVIEW

**Comment by thiago-rocha** [comment]
ci:retest

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-text/text_field.js [hunk 1]
```diff
@@ -169,7 +169,31 @@ describe(
...
+									displayStyle: 'multiline'
+								}
+							}
+						).render(document.body);
+
+						var textareaHeight = textField.getInputNode().get('offsetHeight');
+
+						textField.getInputNode().focus();
+
+						textField.syncInputHeight();
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-text/text_field.js]: Isn't the regression test only supposed to focus the input and check if the height doesn't change?

Comment (thiago-rocha) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-text/text_field.js]: Could you please elaborate more on this question?

**Comment by thiago-rocha** [comment]
ci:retest

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
Pull request submitted to https://github.com/vicnate5/liferay-portal/pull/591. [See changes here.](https://github.com/brunobasto/liferay-portal/compare/e3623f2~0...e3623f2) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
Pull request submitted to https://github.com/vicnate5/liferay-portal/pull/595. [See changes here.](https://github.com/brunobasto/liferay-portal/compare/90e198a~1...90e198a) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**